src/components/Filter/FilterInput.vue

```diff
@@ -447,7 +451,7 @@ $input-height: rem(28px);
     position: relative;
     z-index: 1;
     cursor: text;
-    margin-left: rem(10px);
+    margin-left: var(--input-horizontal-padding);
```

Comment:
maybe not call it padding if used in a margin?

```suggestion
    margin-left: var(--input-horizontal-spacing);
```